index.html

```diff
@@ -1245,7 +1243,7 @@ <h2 id="docconformance">
           </tr>
           <tr id="el-img-no-alt" tabindex="-1">
             <th>
-              [^img^] without <code><a data-cite="html/images.html#unknown-images">alt</a></code> attribute
+              [^img^] without [^img/alt^] attribute
```

Comment:
this changes the link.
now goes to: https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-alt
but should go to https://html.spec.whatwg.org/multipage/images.html#unknown-images

this needs to be fixed

Reply:
The #unknown-images link is not exported, so we can't link to it unfortunately. It would also be inconsistent to link to  link to [^img/alt^] in some places but link to somewhere else to others. 

Let me see if I can find something else to link to, but I think [^img/alt^] is also correct here.

Reply:
agree with your point on having `alt` alone not link to different places.  so i would suggest the link text become "without `alt`" so that it can continue to link to that content.

Reply:
updated link text so as to keep original link